src/Domain/class-wc-email-customer-new-account-blocks.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Class WC_Email_Customer_New_Account_Blocks file.
+ *
+ * @package  WooCommerce/Blocks
+ */
+
+use WC_Email;
```

Comment:
convention is to include `\WC_Email` in imports (root namespace)

Reply:
Some feedback on this class:

- Don't use the old loading process used in Core. Include proper namespacing, and CamelCase for the class name. For now, let's include this `src/Domain/Services/Email` - so the namespace would be `Automattic\WooCommerce\Blocks\Domain\Services\Email` and the fully qualified class name would be `Automattic\WooCommerce\Blocks\Domain\Services\Email\CustomerNewAccount`.
- Instead of the _file_ instantiating the class (when included), you can import this class in `CreateAccount` and then just instantiate it when assigning to the `new_account_email` property.
- With the above you can also drop the `class_exists` check.

Reply:
✨ 41824a1e268fd8599b0c90c696d58da56be2fe37